screens/popups/pick_name/pick_name.gui_script

```diff
@@ -79,8 +79,6 @@ local function okay()
 		end
 		profiles.set_active(profile.slot)
 		network.load_profile(profiles.get_active())
```

Comment:
I would love to see all these other load calls somehow triggered BY the `set_active` call instead of this gui_script doing it. Then we could get rid of the `set_active_complete` nonsense.